src/parser/bison_parser.y

```diff
@@ -508,6 +508,10 @@ create_statement : CREATE TABLE opt_not_exists table_name FROM IDENTIFIER FILE f
   $$->tableName = $4.name;
   $$->setColumnDefsAndConstraints($6);
   delete $6;
+  if (result->errorMsg()) {
+    delete $$;
+    YYERROR;
+  }
```

Comment:
Isn't the `%destructor { delete ($$); } <*>` sufficient here for deallocating the memory?

Reply:
If we omit L512, the memory leak check fails.

Reply:
Okay. Seeing this reminds me of #55 and that addressing it might clean up the code.